test/TestCaseReader.h

```diff
@@ -32,7 +42,7 @@ class TestCaseReader
 	TestCaseReader() = default;
 	explicit TestCaseReader(std::string const& _filename);
 
-	std::map<std::string, std::string> const& sources() { return m_sources; }
+	SourceMap const& sources() { return m_sources; }
```

Comment:
```suggestion
	SourceMap const& sources() const { return m_sources; }
```

The functions below could be `const` too. None of them seems to modify any members and they return `const` values or copies.

Reply:
What about the functions below? Could you change them too? Or was there just something preventing that change?

Reply:
No, there was nothing preventing the change. I may have overlooked your second comment though.